.github/workflows/main.yml

```diff
@@ -15,6 +15,7 @@ concurrency:
 jobs:
     test:
         name: Test on ${{ matrix.platform.os }} using Xcode ${{ matrix.xcode }}
+        environment: ${{ github.event.pull_request.head.repo.fork && 'external' || 'internal' }}
```

Comment:
Why is this needed?

Reply:
I had added it previously, when we have introduced two different environments for secrets as part of using the `authorize` step. 

Removed it now, as we are no longer using `authorize`, please take a look again.